packages/graphql-auth-transformer/src/resources.ts

```diff
@@ -813,11 +816,13 @@ identityClaim: "${rule.identityField || rule.identityClaim || DEFAULT_IDENTITY_F
     // A = IsStaticallyAuthed
     // B = AuthConditionIsNotNull
     // ! (A OR B) == (!A AND !B)
-    public throwIfNotStaticGroupAuthorizedOrAuthConditionIsEmpty(): Expression {
+    public throwIfNotStaticGroupAuthorizedOrAuthConditionIsEmpty(fieldName?: string): Expression {
+        const staticGroupAuthorizedVariable = fieldName ? `${fieldName}_${ResourceConstants.SNIPPETS.IsStaticGroupAuthorizedVariable}` :
```

Comment:
can we move this to a helper function

Reply:
added a helper function in latest commit